src/components/features/CartPopup/CartPopup.js

```diff
@@ -0,0 +1,116 @@
+                    <Button variant='small' className={styles.delete}>
+                      <FontAwesomeIcon icon={faTimesCircle}></FontAwesomeIcon>
+                    </Button>
+                    <img
+                      className={styles.image}
+                      src='https://www.ikea.com/images/dining-sets-03d1940e32032e44c82af0c827a2494a.jpg?f=s'
+                      alt='productFurniture'
+                    />
+                    <span className={styles.productName}>Aean Ru Brisique 14</span>
+                    <span className={styles.price}>$30.00</span>
+                    <div className={styles.quantity}>
+                      <Button variant='outline'>
+                        <span>-</span>
+                      </Button>
+                      <input
+                        type='text'
+                        defaultValue='2'
+                        className={styles.quantityNumber}
+                      ></input>
+                      <Button variant='outline'>
+                        <span>+</span>
+                      </Button>
+                    </div>
+                  </div>
+                  <div className={styles.cartProduct}>
+                    <Button variant='small' className={styles.delete}>
+                      <FontAwesomeIcon icon={faTimesCircle}></FontAwesomeIcon>
+                    </Button>
+                    <img
+                      className={styles.image}
+                      src='https://www.ikea.com/images/dining-sets-03d1940e32032e44c82af0c827a2494a.jpg?f=s'
+                      alt='productFurniture'
+                    />
+                    <span className={styles.productName}>Aean Ru Brisique 14</span>
+                    <span className={styles.price}>$30.00</span>
+                    <div className={styles.quantity}>
+                      <Button variant='outline'>
+                        <span>-</span>
+                      </Button>
+                      <input
+                        type='text'
+                        defaultValue='2'
+                        className={styles.quantityNumber}
+                      ></input>
+                      <Button variant='outline'>
+                        <span>+</span>
+                      </Button>
+                    </div>
+                  </div>
+                </div>
+                <div className={styles.cartSummary}>
+                  <span>Total:</span>
+                  <span>$60.00</span>
+                </div>
+                <div className={styles.buttons}>
+                  <Button variant='small' className={styles.button}>
+                    View cart
+                  </Button>
+                  <Button variant='small' className={styles.button}>
+                    Checkout
+                  </Button>
+                  <Button
+                    variant='small'
+                    className={styles.button}
+                    onClick={this.handleClose}
```

Comment:
Twój kod odwołuje się do funkcji this.handleClose.. nie masz takiej w kodzie. Uruchomienie tego przycisku wywali aplikację.

Jeśli onClick jest koniecznością, to może pusta funkcja:
`() => {}`